Libraries/LibJS/Tests/builtins/Array/Array.from.js

```diff
@@ -50,8 +50,9 @@ describe("normal behavior", () => {
                     let value = begin - 1;
                     return {
                         next() {
-                            if (value < end)
+                            if (value < end) {
                                 value += 1;
+                            }
```

Comment:
Or if you prefer a more linear-looking flow, `value += value < end` or `value += value < end ? 1 : 0`. Feel free to ignore this comment if you like it the way it is.